test/binCases/stats/env/webpack.config.js

```diff
@@ -0,0 +1,11 @@
+var path = require("path");
+
+module.exports = {
+	entry: path.resolve(__dirname, "./index"),
+	stats: {
```

Comment:
Could you clone this test but pass `stats.env: false`? I guess this doesn't work.